ui/components/component-library/text/text.scss

```diff
@@ -33,17 +25,6 @@ $brand-evolution-font-families: (
   }
 }
 
-// Brand evolution font mixin
-@mixin brand-evolution-font($type) {
-  $font-family: map-get($brand-evolution-font-families, $type);
-
-  font-family: $font-family, var(--font-family-sans);
-
-  @if $type == 'display' or $type == 'heading' {
-    font-weight: var(--font-weight-normal);
-  }
-}
```

Comment:
Removing the mapping of different font families to variants as we are using `CentraNo1` for all variants